src/Sulu/Bundle/CustomUrlBundle/Resources/public/scss/main.scss

```diff
@@ -0,0 +1,4 @@
+@import 
+    '../../../../AdminBundle/Resources/public/scss/variables.scss',
```

Comment:
That's something we also do in other bundles, right? Will probably make troubles once we do the subtree split, because the directory structure might change then... But definitely out of scope of this PR.